### ISSUE TITLE
why in model.train() can calculate loss,  model.eval() can not in mask rcnn  torchvision_finetuning_instance_segmentation.ipynb

### ISSUE THREAD

**Issue by Ericargus** [pytorch/vision#1082]

**Comment by fmassa** [comment]
This is how the model was implemented, in order to keep the API simple.
